mica/archive/aca_l0.py

```diff
@@ -144,6 +145,97 @@ def get_options():
     return opt
 
 
+def get_aca_images(start: CxoTimeLike, stop: CxoTimeLike, imgsize=None, columns=None):
+    """
+    Get ACA images in a table from mica l0 archive for a given time range.
+
+    This is a convenience method to get a table of images that should be very similar to
+    that from maude_decom.get_aca_images. These mica images are centered in the 8x8 images
+    via the centered_8x8=True option to aca_l0.get_slot_data to match the maude images.
+
+    Parameters
+    ----------
+    start : CxoTimeLike
+        Start time of the time range.
+    stop : CxoTimeLike
+        Stop time of the time range.
+    imgsize : list of int, optional
+        List of integers of desired image sizes (passed to get_slot_data which defaults
+        to [4, 6, 8] if None).
+    columns : list of str, optional
+        List of desired columns in the ACA0 telemetry. If None defaults to a reasonable
+        minimal list ('TIME', 'QUALITY', 'MJF', 'MNF', 'IMGFUNC1', 'IMGSIZE',
```

Comment:
Let's just change this to all the columns.

Reply:
Done